admin/src/main/resources/io/buoyant/admin/README.md

```diff
@@ -32,3 +32,13 @@ when a file changes:
 ```
 karma start --autoWatch=true --singleRun=false
 ```
+
+## Handlebars templates
+
+For a slight speed boost we precompile handlebars templates. To add a new
+template, simply add the template in templates/ and then run
+
+```
+# npm install handlebars if needed
```

Comment:
this instruction should just be `npm install` since handlebars is listed in devdeps, and it doesn't hurt to run `npm install` even if you're uptodate